### PR TITLE
Deactivate emojis, etc. for Rapid Test Profile

### PR DESCRIPTION
**Pull request by freshking**
## Description

Prevent emoji character input

## Link to Jira

https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7147

Also fixes: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8185

### CODE REVIEW

**Review by whiskey — COMMENTED** [review]
The 🌍  needs more emojis ⚠️

**Comment by freshking** [comment]
> The 🌍 needs more emojis ⚠️

I agree! 👏

**Review by 30mar — APPROVED** [review]

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/CreateAntigenTestProfile/CreateAntigenTestProfileViewController.swift [hunk 1]
```diff
@@ -136,6 +142,13 @@ class CreateAntigenTestProfileViewController: UITableViewController, FooterViewH
 		return true
 	}
 	
+	func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+		return string.trimmingCharacters(in: CharacterSet.alphanumerics).isEmpty
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/CreateAntigenTestProfile/CreateAntigenTestProfileViewController.swift:146]: Can we have a comment here, why we did this, so new developers can understand.

Comment (freshking) [reply src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/CreateAntigenTestProfile/CreateAntigenTestProfileViewController.swift:146]: Good idea

**Review by ArturFriesen — APPROVED** [review]

**Pull request merged**